staging/src/k8s.io/client-go/tools/clientcmd/validation.go

```diff
@@ -30,7 +30,7 @@ import (
 
 var (
 	ErrNoContext   = errors.New("no context chosen")
-	ErrEmptyConfig = NewEmptyConfigError("no configuration has been provided, try setting KUBERNETES_MASTER environment variable")
+	ErrEmptyConfig = NewEmptyConfigError("no configuration has been provided, try setting KUBERNETES_CONTROL_PLANE environment variable")
```

Comment:
@soltysh @deads2k I thought we stopped recommending/documenting this, since the envvar was already marked as deprecated.

Reply:
We never did, but that env is deprecated, unfortunately there are some problems preventing us from entirely dropping this. I'm definitely in favor of changing this comment to mention kubeconfig